ov/hypervisor_cluster_profile.go

```diff
@@ -71,6 +73,7 @@ type HypervisorClusterSettings struct {
 }
 
 type HypervisorHostProfileTemplate struct {
+	DataStoreNameSync         bool                       `json:"dataStoreNameSync,omitempty"`         //"dataStoreNameSync"
```

Comment:
try to keep these boolean values as pointers because in terraform by default will try to make it false, and, therefore might cause idempotency issues.

Reply:
done